### ISSUE TITLE
Support Shelly Motion

### ISSUE THREAD

**Issue by andker87** [yaourdt/mgos-to-tasmota#54]
it would be great to be able to "unlock" shelly motion for use with tasmota and other firmwares.
the great advantage would be to be able to use some devices that are locked to the master firmware because of a "failed" version that does not allow in any way the upgrade

the reference firmware is this:
"SHMOS-01":{"url":"http:\/\/shelly-api-eu.shelly.cloud\/firmware\/SHMOS-01_build.gbl","version":"20210927-192701\/v2.0.1@bd17709f"}

I am available for any information or test

**Comment by MR91** [comment]
I agree with @andker87, Tasmota-Support for Shelly Motion would be awesome. Current FW is 

"SHMOS-01":{"url":"http:\/\/shelly-api-eu.shelly.cloud\/firmware\/SHMOS-01_build.gbl","version":"20211223-144928\/v2.0.5@3f0fcbbe"}
